app/serializers/api/credit_card_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class Api::CreditCardSerializer < ActiveModel::Serializer
+  attributes :id, :month, :year, :cc_type, :last_digits, :first_name, :last_name, :formatted
```

Comment:
If we're not actually going to use `:month, :year, :cc_type, :last_digits, :first_name and :last_name` in the view, we can just make this line:

```ruby
attributes :id, :formatted
```